src/js/tech/html5.js

```diff
@@ -217,22 +237,36 @@ class Html5 extends Tech {
           !elTracks.addEventListener) {
         return;
       }
-      const listeners = {
-        change(e) {
+
+      const listeners = {};
+
+      listeners.change = (e) => {
+        if (!this[`forceNative${props.capitalName}Override_`]) {
           techTracks.trigger({
             type: 'change',
             target: techTracks,
             currentTarget: techTracks,
             srcElement: techTracks
           });
-        },
-        addtrack(e) {
+        }
+      };
+
+      listeners.addtrack = (e) => {
+        if (this[`forceNative${props.capitalName}Override_`] && elTracks.addTrack) {
+          elTracks.addTrack(e.track);
+        } else if (!this[`forceNative${props.capitalName}Override_`]) {
           techTracks.addTrack(e.track);
-        },
-        removetrack(e) {
+        }
+      };
+
+      listeners.removetrack = (e) => {
+        if (this[`forceNative${props.capitalName}Override_`] && elTracks.addTrack) {
+          elTracks.removeTrack(e.track);
```

Comment:
so, I think we don't want to do this part of the if statement for `addTrack` and `removeTrack`, since the event comes from the native elTracks object already.
I think best case scenario is that we should be removing the listeners when we disable native support.